frontend/components/RunCommand.jsx

```diff
@@ -0,0 +1,77 @@
+import { Box } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import { Command } from './CommandWidgets';
+import LoadChartWidget from './LoadChartWidget';
+
+const uptime_cmd = 'uptime';
+
+const graph_title = 'Load average';
+
+export default function RunCommand(props) {
+  const [state, setstate] = useState({
```

Comment:
We use camelCase elsewhere, please try to follow the existing code style (`setState`)